app/(main)/league/[leagueId]/entry/all/page.test.tsx

```diff
@@ -83,28 +85,48 @@ describe('League entries page (Entry Component)', () => {
   });
 
   test('should not display GlobalSpinner after data is loaded', async () => {
+    mockUseDataStore.mockReturnValue({
+      ...mockUseDataStore(),
+      currentWeek: 1,
+      NFLTeams: [
+        {
+          teamId: '1',
+          teamLogo: '/packers-logo.png',
+          teamName: 'Packers',
+        },
+        {
+          teamId: '2',
+          teamLogo: '/bears-logo.png',
+          teamName: 'Bears',
```

Comment:
Future ticket/requires discussion: Relating to my comment previously about the naming conventions, this is inconsistent with the values of `teamId` in `NFLTeams` in [Week.test.tsx](https://github.com/LetsGetTechnical/gridiron-survivor/pull/530/files#diff-aa21d571136ecc543482b81d9dfd60659fbe8af3f789c2028b2e4bd72e404058R82) despite their key values being the same value, `teamId`, leading to some inconsistent and confusing logic within the code itself.

Reply:
@choir27 agreed.  ✅ Updated to be consistent.